libcodechecker/server/api/report_server.py

```diff
@@ -806,20 +806,12 @@ def getRunResults(self, run_ids, limit, offset, sort_types,
 
             filter_expression = process_report_filter(session, report_filter)
 
-            path_len_q = session.query(BugPathEvent.report_id,
-                                       func.count(BugPathEvent.report_id)
-                                       .label('path_length')) \
-                .group_by(BugPathEvent.report_id) \
-                .subquery()
-
             is_unique = report_filter is not None and report_filter.isUnique
             if is_unique:
                 sort_types, sort_type_map, order_type_map = \
                     get_sort_map(sort_types, True)
 
-                selects = [func.max(Report.id).label('id'),
-                           func.min(path_len_q.c.path_length)
-                               .label('bug_path_length')]
+                selects = [func.max(Report.id).label('id')]
```

Comment:
Some sorting code or something disappeared here. Will sorting by BPL still work, or was that always a clientside thing on the webapp?